src/command_modules/azure-cli-interactive/azclishell/app.py

```diff
@@ -479,18 +479,24 @@ def handle_jmespath_query(self, text, continue_flag):
             else:
                 input_dict = self.last.result
             try:
+                cmd_base = text.partition(SELECT_SYMBOL['query'])[0]
```

Comment:
I don't know how partition works, but does it know not to split, for example, on a ? contained in a quoted (or unquoted for that matter) URL?